src/data/invalid/MassSpectrometry-incl-nucleotideseq-slots.yaml

```diff
@@ -0,0 +1,18 @@
+id: nmdc:dgms-99-9n9n9n
+add_date: 30-OCT-14 12.00.00.000000000 AM
+mod_date: 22-MAY-20 06.13.12.927000000 PM
+name: Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D
+processing_institution: JGI
+type: nmdc:MassSpectrometry
+alternative_identifiers:
```

Comment:
Invalid data files should only have one invalid slot at a time. This has a value for alternative_identifiers that doesn't make sense and has both target_gene and ncbi_project_name.

Reply:
The rest of the PR looks good to me.

Reply:
Fixed, thanks Alicia!